README.md

```diff
@@ -1,20 +1,22 @@
-# lando-ui
+# Lando-UI
```

Comment:
nit: Our general convention is `lando-ui` (all lowercase) and `Lando UI` (proper capitalization with a space, no dash).

```suggestion
# Lando UI
```